Sources_v3/StreamChatUI/ChatChannel/MessageComposerInputAccessoryViewController.swift

```diff
@@ -89,11 +94,34 @@ open class MessageComposerInputAccessoryViewController<ExtraData: UIExtraDataTyp
     // MARK: Suggestions
     
     func showSuggestionsViewController() {
-        addChild(suggestionsViewController)
-        view.addSubview(suggestionsViewController.view)
-        suggestionsViewController.didMove(toParent: self)
+        guard let parent = parent else { return }
+        parent.addChild(suggestionsViewController)
+        parent.view.addSubview(suggestionsViewController.view)
+        suggestionsViewController.didMove(toParent: parent)
+
+        guard let suggestionView = suggestionsViewController.view else { return }
+        NSLayoutConstraint(
+            item: suggestionView,
+            attribute: .bottom,
+            relatedBy: .equal,
+            toItem: composerView,
+            attribute: .top,
+            multiplier: 1,
+            constant: 0
+        )
+        .isActive = true
+        NSLayoutConstraint(
+            item: suggestionView,
+            attribute: .centerX,
+            relatedBy: .equal,
+            toItem: composerView,
+            attribute: .centerX,
+            multiplier: 1,
+            constant: 0
+        )
+        .isActive = true
```

Comment:
Anchors API is a bit less nasty I'd say 😄

Reply:
Yeah 😅 Thanks, changed.